core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -1399,7 +1402,7 @@ class LogCleanerTest {
       messageCount += 1
     }
     for (k <- 1 until messageCount by 2)
-      offsetMap.put(key(k), Long.MaxValue)
+      offsetMap.put(key(k), largeDeleteHorizon)
```

Comment:
No need for this change since the intention is to put MaxValue as the offset in the map.